features/polyExplorer/src/components/clusterStories/dataTypes.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState, useContext } from "react";
+
+import { ExplorerContext } from "../../context/explorer-context.jsx";
+import SectionTitle from "./sectionTitle.jsx";
+import MatrixBubblesChart from "./MatrixBubblesChart.jsx";
+import SourceInfoButton from "../sourceInfoButton/sourceInfoButton.jsx";
+import { Tabs, Tab, PolyChart } from "@polypoly-eu/poly-look";
+import {
+    createDataTypesTabs,
+    createDataTypesSharedCombined,
+} from "../../screens/stories/story-utils.js";
+import i18n from "../../i18n.js";
+
+export default function DataTypes({ entities, i18nHeader }) {
+    const { globalData } = useContext(ExplorerContext);
+    const bubbleColor = "#FB8A89";
+    const bubbleStroke = "none";
+    const bubbleTextColor = "#0f1938";
+
+    const listOfDataCategories = Object.keys(
+        globalData.personal_data_categories
+    );
+
+    const dataTypesSharedCombined = createDataTypesSharedCombined(
+        entities,
+        listOfDataCategories
+    );
+
+    const dataTypes = createDataTypesTabs(
+        entities,
+        i18nHeader,
+        listOfDataCategories,
+        dataTypesSharedCombined
+    );
+
+    const [selectedDataTypeBubble, setSelectedDataTypeBubble] = useState(
+        dataTypesSharedCombined[0]["dpv:Category"]
+    );
+
+    const handleBubbleClick = (_, node) => {
+        setSelectedDataTypeBubble(node.data.type);
+    };
+
+    const showLabel = (bubble) => {
+        if (selectedDataTypeBubble === bubble.data.type) {
+            return bubble.data.type
+                .replace("dpv:", "")
+                .replace(/([a-z])([A-Z])/, "$1 $2");
+        }
+    };
+
+    return (
+        <>
+            <SectionTitle
```

Comment:
@RichardSto I added the SectionTitle and the description paragraph in the component because it could be reused, but I'm not sure about this decision since the rest of SectionTitles are in the `messengerStory.jsx` and `digitalGiantStory.jsx`. Do I move them back to the story components?

Reply:
Yeah let's rather keep it uniform, while it makes sense what you say